tests/src/test/java/de/sovity/edc/e2e/PlaceholderDataSourceExtensionTest.java

```diff
@@ -37,19 +37,18 @@
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicReference;
 
-import static de.sovity.edc.extension.e2e.extension.Helpers.defaultE2eTestExtension;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.eclipse.edc.spi.CoreConstants.EDC_NAMESPACE;
 
 class PlaceholderDataSourceExtensionTest {
 
     @RegisterExtension
-    private static E2eTestExtension e2eTestExtension = defaultE2eTestExtension();
+    private static CeE2eTestExtension e2eTestExtension = new CeE2eTestExtension();
 
     @SneakyThrows
     @Test
     void shouldAccessDummyEndpoint(
-        E2eScenario scenario,
+        ApiWrapperConnectorRemote scenario,
```

Comment:
will be renamed to E2eScenario afterwards now that the package structure has become clearer